src/MockClassifier.Api/Models/MessagesInput.cs

```diff
@@ -2,6 +2,7 @@
 {
     public class MessagesInput
     {
-        public List<string> Messages { get; set; }
+        public string CallbackUri { get; set; }
```

Comment:
Just a little question here, might be handled already so apologies if this is redundant, but is there validation on properties in objects throughout the project like here for example we can an attribute like [RegularExpression(@"uri")]?

Reply:
Validation will likely be handled by a future PR. I added in this property so others know that this is a required value in the Classifier contract.